settings/base.py

```diff
@@ -49,7 +61,7 @@ def path(*args):
 STATIC_URL = '/static/'
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = '0D8AE44F-5714-40EF-9AC8-4AC6EB556161'
+SECRET_KEY = config('SECRET_KEY')
```

Comment:
I went ahead and added the `env-dist` file to this PR and updated the (woefully out of date) install docs.